pkg/api/handlers/libpod/images.go

```diff
@@ -186,6 +193,118 @@ func ImagesImport(w http.ResponseWriter, r *http.Request) {
 }
 
 func ImagesPull(w http.ResponseWriter, r *http.Request) {
-	//TODO ...
-	utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.New("/libpod/images/pull is not yet implemented"))
+	runtime := r.Context().Value("runtime").(*libpod.Runtime)
+	decoder := r.Context().Value("decoder").(*schema.Decoder)
+	query := struct {
+		Reference    string `schema:"reference"`
+		Credentials  string `schema:"credentials"`
+		OverrideOS   string `schema:"override-os"`
```

Comment:
camelCase is preferred to dashes, to save on length.  eg., overrideOs vs override-os

Reply:
Good point, will do!